### PR TITLE
change to TypeScript

### PR DESCRIPTION
**Pull request by taka-sho**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import path from 'path'
```
Comment (9renpoto) [inline webpack.config.js]: use `webpack.config.ts`

File: __tests__/options.spec.tsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+declare var React: any
+
 describe('options', () => {
   const data = {
     id: 'id',
     env: 'env',
     host: 'host',
-    ignore: 'ignore'
+    ignore: 'ignore',
+    isActive: !!'isActive'
```
Comment (9renpoto) [inline __tests__/options.spec.tsx]: 🤔

File: app/components/options.tsx [hunk 3]
```diff
@@ -1,21 +1,17 @@
-/* @flow */
-/** @jsx h */
-import { h, Component } from 'preact'
+import { Component, h } from 'preact'
 import { set } from '../actions/options'
 
+declare var React: any
```
Comment (9renpoto) [inline app/components/options.tsx]: Is it necessary?

File: __tests__/injector.spec.ts [hunk 4]
```diff
@@ -11,7 +10,7 @@ describe('injector', () => {
       host: random.alphaNumeric(15),
       ignore: random.alphaNumeric(15)
     }
-    inject(elementId, random.uuid(), config)
+    inject(elementId, `a${random.uuid()}`, config)
```
Comment (taka-sho) [inline __tests__/injector.spec.ts]: it must start as alphabet......

File: package.json [hunk 5]
```diff
@@ -47,29 +44,38 @@
...
   },
   "keywords": [],
   "lint-staged": {
-    "*.{js,jsx}": [
+    "*.{ts,tsx}": [
       "prettier-eslint --write",
-      "eslint --cache",
-      "git add"
-    ],
-    "*.css": [
```
Comment (9renpoto) [inline package.json]: please revert it

**Pull request merged**